src/graph/context/Result.h

```diff
@@ -63,6 +63,15 @@ class Result final {
     }
   }
 
+  std::vector<std::string> getColNames() const {
```

Comment:
`const std::vector<std::string>& getColNames() const`

Reply:
NCK.
Cannot return reference if data type is not dataset.